src/core/view/view.ts

```diff
@@ -101,6 +109,22 @@ export abstract class View {
         });
     }
 
+    public setAutomaticLayout(enable: boolean): void;
```

Comment:
Not sure how you feel about overloading in Typescript, again I pretty much copied the Java API.